class_configs/pal_class_config.lua

```diff
@@ -655,32 +655,41 @@ return {
             {
                 name = "WaveHeal",
                 type = "Spell",
-                cond = function(self, _) return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end,
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
+                cond = function(self, aaName)
+                    if mq.TLO.Group() then return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end
+                end,
             },
             {
                 name = "WaveHeal2",
                 type = "Spell",
-                cond = function(self, _) return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end,
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
+                cond = function(self, aaName)
+                    if mq.TLO.Group() then return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end
```

Comment:
these should all have an else return false.

Reply:
Added